doc/installation.rst

```diff
@@ -0,0 +1,267 @@
+Installation Guide
+==================
+
+This guide covers the installation and setup of the FoamAdapter Python module, which provides a Python interface to OpenFOAM solvers and utilities through the pybFoam backend.
+
+Prerequisites
+-------------
+
+System Requirements
+~~~~~~~~~~~~~~~~~~~
+
+Before installing FoamAdapter Python module, ensure your system meets the following requirements:
+
+* **Operating System**: Linux (Ubuntu 20.04+, CentOS 8+, or similar)
+* **Python**: 3.9 or higher
+* **OpenFOAM**: Version 2406 or later
+* **CMake**: 3.22 or higher
+* **Compiler**: GCC 12+ or Clang 18+
+* **CUDA**: 12.1+ (optional, for GPU acceleration)
```

Comment:
We should add rocm here, because that is already tested on neon side. and ideally sycl soon.